docs/i18n.md

```diff
@@ -27,47 +28,122 @@ NODE_APP_INSTANCE=[MY_APP] NODE_PATH='./:./src' bin/create-locales
+**If no strings were updated then you do not have to continue creating the pull
+request. You can revert the changes made to the `pot` timestamp.**
+Here is an example of a diff where no strings were changed. It just shows
+a single change to the timestamp:
+
+```diff
+diff --git a/locale/templates/LC_MESSAGES/amo.pot b/locale/templates/LC_MESSAGES/amo.pot
+index 31e113f2..c7da4e34 100644
+--- a/locale/templates/LC_MESSAGES/amo.pot
++++ b/locale/templates/LC_MESSAGES/amo.pot
+@@ -2,7 +2,7 @@ msgid ""
+ msgstr ""
+ "Project-Id-Version: amo\n"
+ "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+-"POT-Creation-Date: 2017-06-08 14:01+0000\n"
++"POT-Creation-Date: 2017-06-08 14:43+0000\n"
+ "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+ "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+ "Language-Team: LANGUAGE <LL@li.org>\n"
+```
+
+When the application is under active development it's more likely that you will
+see a diff containing new strings or at least strings that have shifted to
+different line numbers in the source. If so, commit your change and continue
+to the next step:
+
+```
+git commit -a "Extract AMO locales"
+```
+
+### Merging locale files
+
+After extracting new strings, you have to merge them into the existing locale
+files. Do this in your branch and commit:
 
 ```
-# First extract the new strings.
-NODE_APP_INSTANCE=[MY_APP] bin/extract-locales
-# Merge the new strings into the locale files.
-NODE_APP_INSTANCE=[MY_APP] bin/merge-locales
+NODE_APP_INSTANCE=amo bin/merge-locales
+```
+
+At this point, you may want to check for
+[fuzzy strings](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html)
+by running `git diff` and searching for a comment that looks like
+`# fuzzy`. This comment means the localization may not exactly match the source text;
+a localizer needs to review it.
+For applications like `disco` (The Discovery Pane) you may not want
+to merge a pull request until the fuzzy strings are resolved.
+
+Commit and continue to the next step:
+
+```
+git commit -a "Merged AMO locales"
 ```
 
 ### Building the debug locales
 
-Using this command Generates a unicode debug locale and a mirror rtl debug
+The next step is to generate a unicode debug locale and a mirror rtl debug
 locale using [potools](https://github.com/mozilla/potools/).
 
-To build the debug locale files run:
+Build the debug locale and commit it to your branch:
+
+```
+NODE_APP_INSTANCE=amo bin/debug-locales
+git commit -a "Generated AMO debug locale"
+```
+
+### Finalizing the extract/merge process
+
+Now that you have extracted, merged, and generated a debug locale for one
+application, it's time to create a pull request for your branch. For example:
 
 ```
-# build the debug locale files.
-NODE_APP_INSTANCE=[MY_APP] bin/debug-locales
+git push origin amo-locales
 ```
 
+Request a review for the pull request and merge it like any other.
+If necessary, repeat the process for the next application.
+
 ### Building the JS locale files
```

Comment:
You could note that this part is now automatically run for deployments and tests.